module/module.go

```diff
@@ -153,13 +153,12 @@ type peerResourceState struct {
 	// NOTE As I'm only suppporting video to start this will always be a single element
 	// once we add audio we will need to make this a slice / map
 	subID rtppassthrough.SubscriptionID
-	// NOTE As I'm only suppporting video to start this will always be a single element
-	// once we add audio we will need to make this a slice
-	sender *webrtc.RTPSender
```

Comment:
We no longer need to keep the `*webrtc.RTPSender` on the `peerResourceState` as it now only needs to live in a closure.